Snippets/Core/Core_6.5/Core_6.5.csproj

```diff
@@ -5,6 +5,9 @@
     <PlatformTarget>x64</PlatformTarget>
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Let's drop this and add `<NoWarn>$(NoWarn);IDE1002</NoWarn>` to the existing PropertyGroup instead for all the places it has been changed.